src/Elastic.CommonSchema.Serilog/LogEventConverter.cs

```diff
@@ -0,0 +1,335 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Security.Principal;
+using System.Threading;
+using System.Web;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Elastic.CommonSchema.Serilog
+{
+    /// <summary>
+    /// Elastic Common Schema converter for LogEvent
+    /// </summary>
+    public class LogEventConverter
+    {
+        public static Base ConvertToEcs(LogEvent e)
+        {
+            var request = HttpContext.Current?.Request;
```

Comment:
HttpContext not .net core friendly. These enrichers need to live on the formatter as options imo

Reply:
I've implemented a configuration object (`descriptor` using the same system and naming as NEST) where you can set which elements of the ECS graph you want enriched. Allows for either full or partial enrichment.